README.md

```diff
@@ -96,8 +96,7 @@ throw new errors.BalenaInvalidDeviceType('raspberry-pi')
 <a name="new_module_errors..BalenaDiscontinuedDeviceType_new"></a>
 
 #### new BalenaDiscontinuedDeviceType(type)
-The device type that you specified is invalid because it is
-discontinued, and this operation is no longer supported.
+The device type that you specified is invalid because it is
discontinued, and this operation is no longer supported.
```

Comment:
Was this `README.md` file accidentally converted to Windows line endings?

Reply:
It would have been on checkout but then back to linux on checkin, my guess is that because it's auto-generated from the source files that the tool doesn't handle windows properly (potentially a bug in the setup from this module, or possibly a bug in the tool - it's probably really out of date)